docker/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  trin:
+    hostname: trin
+    image: portalnetwork/trin:latest
+    environment:
+      RUST_LOG: info
+    command: "--web3-transport http --web3-http-address http://0.0.0.0:8545/"
+    network_mode: "host"
+    volumes: 
+      - $HOME/.local/share:$HOME/.local/share
+    ports:
+      - "8545:8545"
```

Comment:
Wouldn't this also need to expose the discovery port, default 9009?

Reply:
Since host mode is enabled I should remove the ports entry as in this case we use the host ports so this entry doesn't do anything

Reply:
Yeah, we don't need the port mapping if "host" is enabled.